libs/s25main/mapGenerator/MapSettings.h

```diff
@@ -31,25 +31,30 @@ namespace rttr { namespace mapGenerator {
         Mixed
     };
 
-    struct MapSettings
+    enum class MountainDistance : uint8_t
     {
-        MapSettings()
-            : numPlayers(2), size(MapExtent::all(128)), ratioGold(9), ratioIron(36), ratioCoal(40), ratioGranite(15),
-              rivers(15), type(0), style(MapStyle::Mixed)
-        {}
+        Close = 5,
+        Normal = 15,
+        Far = 25,
+        VeryFar = 30
```

Comment:
I'm not sure if it is obvious why I chose those numbers. In theory, you might think you would like this to depend on the map size. In practice, it can get really annoying if you play a 1024x1024 map and you're 200 nodes away from any mountain. You might never reach it. 

So I thought, even on larger maps it's still good to be in reach of mountains within a reasonable amount of time. The "reasonable amount of time" would be equal, independent of the map size.